app/src/main/java/org/jellyfin/androidtv/ui/SettingsPopup.kt

```diff
@@ -0,0 +1,73 @@
+package org.jellyfin.androidtv.ui
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.PopupWindow
+import org.jellyfin.androidtv.databinding.SettingsPopupBinding
+import org.jellyfin.androidtv.util.Utils
+
+class SettingsPopup(
+	context: Context,
+	anchor: View,
+	audioChangedListener: ValueChangedListener<Long>?,
+	subtitleChangedListener: ValueChangedListener<Long>?
+) {
+	val popupWindow: PopupWindow?
+	private val anchor: View
+	private val binding: SettingsPopupBinding
+
+	init {
+		val width = Utils.convertDpToPixel(context, width)
+		val height = Utils.convertDpToPixel(context, fullHeight)
+
+		binding = SettingsPopupBinding.inflate(LayoutInflater.from(context), null, false)
+
+		popupWindow = PopupWindow(binding.root, width, height)
+		popupWindow.isFocusable = true
+		popupWindow.isOutsideTouchable = true
+		popupWindow.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT)) // necessary for popup to dismiss
+
+		this.anchor = anchor
```

Comment:
You can add `private val` to the variable in the constructor instead to make it a class property.